### DISCUSSION TITLE
4G Decoding cant be found in the extracted file by irfextractor

### DISCUSSION THREAD

**Discussion by MmdJoyeuse** [xCuri0/ReBarUEFI#74] [Q&A] [answered]
Hello :)
So i modified the bios and disabled CSM.
There is no 4G Decoding in bios so i should use the "Enable hidden 4G decoding "
the thing is, **i have 4g decod in uefitool search**. but there is no such thing in the **extracted file**.
btw my system is: 
MB: msi h81 p33
CPU: i5 4570 which is haswell
GPU: rx580 8gb

**Comment by xCuri0** [comment, marked as answer]
https://github.com/xCuri0/ReBarUEFI/issues/14

if you use linux you can make it work with dsdt edit
